parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -205,6 +205,38 @@ downloadContainerdWasmShims() {
     done
 }
 
+# most logic duplicated from downloadContainerdWasmShims
+downloadContainerdWasmShimsFromRegistry() {
```

Comment:
Since a lot of this is duplicated, it might make more sense to to do 
`if [[ ! -z ${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER} ]]; then` inside downloadContainerdWasmShims and just have one function. Similair to how I did credential provider.

Reply:
removed and merged with `downloadContainerdWasmShims`